spec/models/expenditure_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  let(:user) { User.create(id: 1, full_name: 'user', email: 'user@mail.com', password: '123456') }
+  let(:group) { Group.create(id: 2, name: 'Shopping', icon: 'Shopping', user_id: user.id) }
+  let(:expenditure) { Expenditure.create(id: 1, name: 'Dress', amount: 12, user_id: user.id, group_id: group.id) }
+
+  describe 'Expenditure validations' do
+    context 'when valid' do
+      it { expect(expenditure).to be_valid }
+    end
+
+    it 'Without name it is invalid ' do
+      expenditure.name = nil
+      expect(expenditure).to_not be_valid
+    end
+  end
+end
```

Comment:
- [ ] Test are failing please check an error you are having in this file
![tets](https://user-images.githubusercontent.com/44745855/159766668-532f31c6-2d70-4e4f-b2c8-1e00a120ee3b.png)

Reply:
- [ ] If you fix the syntax error in the expenditure model, model tests will pass. Please fix it.